ci/edmtool.py

```diff
@@ -262,8 +262,10 @@ def test(runtime, toolkit, pillow, environment):
     environ = environment_vars.get(toolkit, {}).copy()
     environ['PYTHONUNBUFFERED'] = "1"
     commands = [
-        "edm run -e {environment} -- coverage run -m nose.core enable -v --nologcapture",
-        "edm run -e {environment} -- coverage run -a -m nose.core kiva -v --nologcapture",
+        "edm run -e {environment} -- python -W default -m"
+        "coverage run -m nose.core enable -v --nologcapture",
```

Comment:
It might be helpful to wrap these multi-line commands in parentheses so that it's clear they belong together. Otherwise you have to be paying attention to the tiny commas that separate the commands.